README.md

```diff
@@ -1 +1,92 @@
-# JSCapstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Js Capstone
+
+> In this project I'm practicing how to use RESTful API's. I'm using GET and POST methods to either display shows on the screen or to Display data that's related to the Involvement API that Microverse built for this learning project.
+
+Basic UI of the project:
+
+![screenshot](./screenshot1.png)
+
+UI of the Pop Up:
+![screenshot](screenshot2.png)
+
+## Built With
+
+- Js
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://nelsinofrancisco.github.io/JSCapstone/)
+
+## Project Description (Loom Video)
+
+Check out the [video](https://www.loom.com/share/9c979ec4ed3a4784b4cafb54013640b7)
+
```

Comment:
- [ ] As requested in the previous review, in the video you should show and speak about a piece of code that you find interesting. Kindly record a new video doing that.

Reply:
I've done that, multiples times actually, but check out in my video the minute 3:16. I know that it's a long video, but I was talking about a part of my code function() that I enjoy doing and what the function is doing too.

Reply:
This issue is well addressed. I also commend you for effective communication by mentioning the time in the video I should look at. :+1: